12C/Homework

```diff
@@ -0,0 +1,9 @@
+clear
+my_number = 22
+t = 0.01:0.01:my_number;
```

Comment:
Да не би да трябва да е clear all

On Mon, Apr 4, 2016 at 12:06 PM, Dimitar Nikolov <notifications@github.com>
wrote:

> In 12C/Homework
> <https://github.com/nikolovd/pps/pull/48#discussion_r58346462>:
>
> > @@ -0,0 +1,9 @@
> > +clear
> > +my_number = 22
> > +t = 0.01:0.01:my_number;
>
> Тук има грешка
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/nikolovd/pps/pull/48/files/d577a6b8b1f2e2577859db7c690f29f3dc009eb4#r58346462>
>